indexer-app/src/cmr/indexer/data/concepts/collection/keyword.clj

```diff
@@ -82,9 +82,14 @@
         instruments (concat platform-instruments (mapcat :composed-of platform-instruments))
         instrument-short-names (distinct (keep :short-name instruments))
         instrument-techiques (keep :technique instruments)
-        characteristics (mapcat :characteristics platforms)
-        char-names (keep :name characteristics)
-        char-descs (keep :description characteristics)
+        plat-characteristics (mapcat :characteristics platforms)
+        plat-char-names (keep :name plat-characteristics)
+        plat-char-descs (keep :description plat-characteristics)
+        plat-char-values (keep :value plat-characteristics)
+        instr-characteristics (mapcat :characteristics instruments)
+        instr-char-names (keep :name instr-characteristics)
+        instr-char-descs (keep :description instr-characteristics)
+        instr-char-values (keep :value instr-characteristics)  
```

Comment:
I sent him an email and he replied. The reason he thinks they are indexed is because our document says so and I think he might just test the platform search.  My test result shows it doesn't work for instrument/child instrument search. so I'm adding them.